dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -324,3 +324,18 @@ def unalikeability(self) -> float | None:
             ) * self._categories[category]
         unalike: float = unalike_sum / (self.sample_size**2 - self.sample_size)
         return unalike
+
+    def to_dict(self) -> dict:
+        """
+        Create serializable dict of this object.
+
+        :return: dict
+        """
+        categorical_dict = {
+            "type": self.type,
+            "_categories": self._categories,
+            "__calculations": self.__calculations,
```

Comment:
I think this will cause issues in the future, but currently is unimportant bc there's no calc to track. We might need a reusable func to serialize and another for later loading funcs. However, we can address this when it shows in subsequent updates.